docs/data/material/components/modal/ServerModal.js

```diff
@@ -13,11 +13,6 @@ export default function ServerModal() {
         flexGrow: 1,
         minWidth: 300,
         transform: 'translateZ(0)',
-        // The position fixed scoping doesn't work in IE11.
```

Comment:
Introduced in https://github.com/mui/material-ui/pull/17972. Rolling back to original.